tests/s3/test_object_tagging.py

```diff
@@ -425,7 +393,7 @@ def test_add10tags_existingobject_2464(self):
                 S3_OBJ_TST["test_9420"]["value"],
                 tag_count=S3_OBJ_TST["test_9420"]["tag_count"])
         except CTException as error:
-            assert errmsg.S3_BKT_INVALID_TAG_ERR in str(error.message), error.message
+            assert errmsg.S3_RGW_BKT_INVALID_TAG_ERR in str(error.message), error.message
```

Comment:
This test will fail for legacy s3 tests.
To make it compatible for both CORTX with legacy S3 and CORTX-RGW, use function assert_s3_err_msg().
Refer file test_bucket_tagging.py: 210

Reply:
Updated to make compatible for both CORTX with legacy S3 and CORTX-RGW.